src/server/db_slice.cc

```diff
@@ -543,6 +556,7 @@ tuple<PrimeIterator, ExpireIterator, bool> DbSlice::AddOrFind2(const Context& cn
     throw e;
   }
 
+  DCHECK(inserted) << "Inserted key " << key << " after not finding it";
```

Comment:
Note that in my suggestion I wrote about Adding a new function to dash table that on Insert will not try to find if the value exists. As in the flow now you do 2 find operations in dash table if this is a new entry

Reply:
Right, I forgot about that. Done now.